classes/sdimg.bbclass

```diff
@@ -0,0 +1,112 @@
+# Class that creates an SD card image that boots under qemu's emulation
+# for vexpress-a9 board. See the script mender-qemu for an example of
+# how to boot the image.
+
+# The partitioning scheme is:
+#    part1: FAT partition with bootloader
+#    part2: first rootfs, active
+#    part3: second rootfs, inactive, mirror of first,
+#           available as failsafe for when some update fails
+#    part4: persistent data partition
```

Comment:
You should write `part5`. `part4` is the extended partition which contains `part5`.